### ISSUE TITLE
multi-line OCR output as single line text

### ISSUE THREAD

**Issue by william5562** [TheJoeFin/Text-Grab#459]
**Describe the bug**
multi-line OCR output as single line text

**Where is the bug**
- Fullscreen Grab "multi-line OCR"
- OCR Output

**Expected behavior**
The multi-line OCR should output as multi-line text.

**Screenshots**
![image](https://github.com/TheJoeFin/Text-Grab/assets/152057435/2e7c312c-29b2-4a87-886f-4abfa7217351)

**Where did you get Text Grab?**
- Microsoft Store

**Desktop (please complete the following information):**
 - OS: [Windows 11]
 - Version [23H2]
 - Text Grab Version [4.4.2]

**Additional context**
- Fullscreen Grab "multi-line OCR" is more accurate than "OCR text as a table" maybe because "OCR as table" isn't using Tesseract engine?             (inside program description: Does not use Tesseract: Table Recogintion and the Grab Frame.)

**Comment by william5562** [comment]
![image](https://github.com/TheJoeFin/Text-Grab/assets/152057435/13e1ce6d-9b2b-46a4-bdc4-6736264f4504)

**Comment by TheJoeFin** [comment]
This option from the Fullscreen Grab makes the output a single line. Are you sure you do not have that selected?

<img width="465" alt="image" src="https://github.com/TheJoeFin/Text-Grab/assets/7809853/2f24d849-7700-418b-9d3e-dd0acbc3cf1f">

**Comment by william5562** [comment]
Yeah, pretty sure I selected multi-line, but language is set to traditional Chinese(Taiwan).

After retry, this issue seem only affect traditional Chinese. English works fine.
